packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/ScopeHandlerFactoryImpl.ts

```diff
@@ -114,4 +115,15 @@ export class ScopeHandlerFactoryImpl implements ScopeHandlerFactory {
           ?.getScopeHandler(scopeType);
     }
   }
+
+  create(
```

Comment:
I would be tempted to call this `createStrict` or `createOrThrow`. Once we've migrated all scopes, this will never throw, right? So maybe we just leave it 🤷‍♂️

Reply:
Yes this shouldn't really throw. It's just there because I don't like doing `!` assertion in typescript if we can get around it.